src/parser.js

```diff
@@ -0,0 +1,7 @@
+const parse = (tokens) => {
```

Comment:
This file should be removed from this PR. Tests fail as there are none for this test suite.